header.go

```diff
@@ -513,6 +514,85 @@ func fixUnquotedSpecials(s string) string {
 	return clean.String()
 }
 
```

Comment:
Could you add a very short description of func with an example so the reader can visualize the transformation it's making?

Reply:
I think GH got confused here, this was meant to go above the func fixUnescapedQuotes definition